wrangler-transform/src/main/java/io/cdap/wrangler/Wrangler.java

```diff
@@ -385,9 +387,11 @@ public void transform(StructuredRecord input, Emitter<StructuredRecord> emitter)
           emitter.emitError(new InvalidEntry<>(error.getCode(), error.getMessage(), input));
           errorMessages.add(error.getMessage());
         }
-        if (onErrorStrategy.equalsIgnoreCase("fail-pipeline")) {
-          throw new Exception(
-            String.format("Errors in Wrangler Transformation - %s", errorMessages));
+        if (WRANGLER_FAIL_PIPELINE_FOR_ERROR.isEnabled(getContext())) {
+          if (onErrorStrategy.equalsIgnoreCase("fail-pipeline")) {
```

Comment:
Can be combined as:
```
if WRANGLER_FAIL_PIPELINE_FOR_ERROR.isEnabled(getContext())
   && onErrorStrategy.equalsIgnoreCase("fail-pipeline") {
    throw new Exception(...)
}
```

Reply:
Makes sense. How are feature flags treated once it is removed? I assumed that if condition block will be removed hence added it in nested way.

Reply:
Updated